### PR TITLE
[RV64_DYNAREC] Added 8B MOV opcode

### PR DESCRIPTION
**Pull request by xctan**

### CODE REVIEW

File: src/dynarec/rv64/rv64_emitter.h [hunk 1]
```diff
@@ -221,6 +221,8 @@ f28–31  ft8–11  FP temporaries                  Caller
 
 // rd = [rs1 + imm12]
 #define LD(rd, rs1, imm12)          EMIT(I_type(imm12, rs1, 0b011, rd, 0b0000011))
+// rd = [rs1 + imm12]
+#define LDxw(rd, rs1, imm12)        EMIT(I_type(imm12, rs1, 0b011<<(1-rex.w), rd, 0b0000011))
```
Comment (ptitSeb) [inline src/dynarec/rv64/rv64_emitter.h:225]: I don't think this is correct. 
it should be `EMIT(I_type(imm12, rs1, 0b010+rex.w, rd, 0b0000011))`

**Review by ptitSeb — CHANGES_REQUESTED** [review]
I don't think `LDwx` emitter is correct.

File: src/dynarec/rv64/rv64_emitter.h (hunk 1 above)
Comment (xctan) [reply src/dynarec/rv64/rv64_emitter.h:225]: If funct3 is 0b010, the instruction would be LW and the result is sign-extended to 64-bits.

Comment (ptitSeb) [reply src/dynarec/rv64/rv64_emitter.h:225]: Yes, indeed, I double checked and you are right, it's better to use LWU.

**Review by ptitSeb — APPROVED** [review]

**Pull request merged**